packer/windows.pkr.hcl

```diff
@@ -245,15 +236,15 @@ build {
         tags = ["${var.image_date}", "latest"]
         # packer version is 1.6.6 while generating the vm images, and it complains
         # if local.only.docker is used here
-        only = ["docker.windows_ci_vs_2019", "docker.windows_ci_mingw64"]
+        only = ["docker.windows_container", "docker.windows_container"]
```

Comment:
Let's drop the windows containers before we get to this bit - since all users would need to change the name of the container anyway, there's not much point in continuing to build them.